Lesson_03/homework_03.py

```diff
@@ -0,0 +1,149 @@
+"""Solutions for homework_03 tasks."""
+
+# alice_in_wonderland = '"Would you tell me, please, which way
+#            I ought to go from here?"\n
+#           "That depends a good deal on where you want to get to,"
+#            said the Cat.\n"I don't much care where ——" said Alice.\n
+#           "Then it doesn't matter which way you go," said the Cat.\n
+#           "—— so long as I get somewhere," Alice added as an explanation.\n
+#           "Oh, you're sure to do that," said the Cat,
+#           "if you only walk long enough."'
+# task 01 == Розділіть змінну alice_in_wonderland так,
+#            щоб вона займала декілька фізичних лінії
+# task 02 == Знайдіть та відобразіть всі символи одинарної лапки (') у тексті
```

Comment:
There was no info from my side about string operations - and probably task description has lack information about what exactly required  - so let's now just "close eyes" 
If required to find all ( maybe they mean to count them) - in this case  for you could be helpful count string method 
`help(str.count)`